renderer/backend/metal/render_pass_mtl.mm

```diff
@@ -434,6 +435,17 @@ static bool Bind(PassBindingsCache& pass,
                               ShaderStage::kFragment)) {
       return false;
     }
+    MTLIndexType index_type;
```

Comment:
I'd rather put this in `formats_mtl.h`. See `ToMTLPrimitiveType(command.primitive_type)` for instance. So something like `ToMTLIndexType(impeller::IndexType)`

Reply:
Alrighty, done.